docs-2.0/3.ngql-guide/6.functions-and-expressions/4.schema.md

```diff
@@ -322,6 +322,32 @@ nebula> MATCH (v:player{name:"Tim Duncan"})-[e]->() \
 +----------+
 ```
 
+### typeid()
+
+typeid() 返回边的 Edge type 的内部 ID 值，可以用正负来判断方向。
+
+语法：`typeid(<edge>)`
+
+- 返回类型：int。
+
+示例：
+
+```ngql
+nebula> MATCH (v:player)-[e:follow]-(v2) RETURN e,typeid(e), \
+        CASE WHEN typeid(e) > 0 \
+        THEN "正向" ELSE "反向" END AS direction \
+        LIMIT 5;
++----------------------------------------------------+-----------+-----------+
+| e                                                  | typeid(e) | direction |
++----------------------------------------------------+-----------+-----------+
+| [:follow "player127"->"player114" @0 {degree: 90}] | 5         | "正向"    |
+| [:follow "player127"->"player148" @0 {degree: 70}] | 5         | "正向"    |
+| [:follow "player148"->"player127" @0 {degree: 80}] | -5        | "反向"    |
+| [:follow "player147"->"player136" @0 {degree: 90}] | 5         | "正向"    |
+| [:follow "player136"->"player147" @0 {degree: 90}] | -5        | "反向"    |
++----------------------------------------------------+-----------+-----------+
+```
+
```

Comment:
Maybe we can link the similar ngql usage of `<edgetype>._type` or reference it in this section.